### ISSUE TITLE
How to make this performs well?

### ISSUE THREAD

**Issue by snippins** [mumu-lhl/eaf-pyqterminal#39]
Hello, I'm on X11, I saw the demo image with btop so I when on and test eaf-pyqterminal with it, however everything is lagging mad, try a few more tui apps like helix, neovim, kakoune, htop and they're all slow? What could be the reasons? What could I possibly did wrong here?

**Comment by mumu-lhl** [comment]
These tui apps have a lot of rendering, and the current EAF PyQterminal rendering is not very efficient, so it will be slow.

No way to improve rendering efficiency has been found yet :(

**Comment by snippins** [comment]
Welp, sorry to hear that, though isn't it funny when you can have a performance browser in eaf but not a performance terminal (yet). Though I imagine the slowness is entirely because of the terminal app and not Emacs right, as eaf is just reparenting the window and emacs is not actually rendering anything?

**Comment by mumu-lhl** [comment]
Yes, it is. The reason why the browser in eaf is faster is because it uses QWebEngineView. But the rendering feature of the terminal app is written in python.

**Comment by snippins** [comment]
@mumu-lhl Have you looked at this? Is it possible to use this lib as the backend?

https://github.com/lxqt/qtermwidget

**Comment by mumu-lhl** [comment]
@snippins I had considered it, but the qtermwidget doesn't have full functionality and is a dumb terminal.
